src/hermes/commands/workflow/__init__.py

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2023 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-FileContributor: David Pape
+
+from .clean import clean
+from .curate import curate
+from .deposit import deposit
+from .harvest import harvest
+from .postprocess import postprocess
+from .process import process
```

Comment:
This seems to confuse Sphinx AutoDoc. Is this propagation required?

Reply:
It is not required. But if we do it like this, we can keep [this block as is](https://github.com/hermes-hmc/workflow/blob/e5ad51d1498f3a82266a8e4b9b35254572506d88/src/hermes/cli.py#L165). The alternative would be to have

```python
main.add_command(workflow.clean.clean)
main.add_command(workflow.harvest.harvest)
main.add_command(workflow.process.process)
main.add_command(workflow.curate.curate)
main.add_command(workflow.deposit.deposit)
main.add_command(workflow.postprocess.postprocess)
```

in the CLI module.

Reply:
I decided not to split the code into multiple files in this PR. So this was removed again.